bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -609,20 +625,34 @@ void BCStateTran::getDigestOfCheckpointImpl(uint64_t checkpointNumber,
   ConcordAssert(psd_->hasCheckpointDesc(checkpointNumber));
 
   DataStore::CheckpointDesc desc = psd_->getCheckpointDesc(checkpointNumber);
-  LOG_INFO(logger_,
-           KVLOG(desc.checkpointNum, desc.maxBlockId, desc.digestOfMaxBlockId, desc.digestOfResPagesDescriptor));
+  auto rvbDataSize = desc.rvbData.size();
+  size_t copysize = std::min((uint16_t)sizeof(Digest), sizeOfDigestBuffer);
```

Comment:
why do we need to downcast the sizeof Digest. Instead we can upcast sizeOfDigestBuffer
auto copysize = std::min(sizeof(Digest), static_cast<size_t>(sizeOfDigestBuffer));

Reply:
That's fine, I agree this one is better. Thanks.